cli/cmd/proxy/read/config.go

```diff
@@ -363,7 +363,7 @@ func formatFilters(filterChain filterChain) (filters []string) {
 }
 
 func formatFilterTCPProxy(config typedConfig) (filter string) {
-	return "-> " + config.Cluster
+	return "to " + config.Cluster
```

Comment:
Changed here and below because the parsing to bytes with the JSON output would print `"->"` as `"-\u003e"`. I figured it was better to just go around the problem rather than try to figure out why bytes wasn't using the `>` character even though it is ASCII. There might be a way to marshal into runes, but this solution was just cleaner.